### PR TITLE
reply_or_reply_to_message and added missed string

### PR DESCRIPTION
**Pull request by iiiiii1wepfj**

### CODE REVIEW

File: eduu/plugins/custom_filters.py [hunk 1]
```diff
@@ -164,6 +164,7 @@ async def get_all_filter(c: Client, m: Message, strings):
 async def serve_filter(c: Client, m: Message):
     chat_id = m.chat.id
     text = m.text
+    reply_or_reply_to_msg = m.reply_to_message if m.reply_to_message else m
```
Comment (AndrielFR) [inline eduu/plugins/custom_filters.py:167]: There is no better name for this variable? Something like `reply_or_message`.

Comment (iiiiii1wepfj) [reply eduu/plugins/custom_filters.py:167]: @AndrielFR idk

Comment (AndrielFR) [inline eduu/plugins/custom_filters.py:167]: `m.reply_to_message or m` is better.

File: eduu/plugins/notes.py [hunk 2]
```diff
@@ -243,7 +243,8 @@ async def serve_note(c: Client, m: Message, txt):
 )
 async def note_by_hashtag(c: Client, m: Message):
     note_data = m.text[1:]
-    await serve_note(c, m, txt=note_data)
+    reply_or_reply_to_msg = m.reply_to_message if m.reply_to_message else m
```
Comment (AndrielFR) [inline eduu/plugins/notes.py:246]: Same here.

File: eduu/plugins/notes.py [hunk 3]
```diff
@@ -255,7 +256,8 @@ async def note_by_hashtag(c: Client, m: Message):
 )
 async def note_by_get_command(c: Client, m: Message):
     note_data = " ".join(m.command[1:])
-    await serve_note(c, m, txt=note_data)
+    reply_or_reply_to_msg = m.reply_to_message if m.reply_to_message else m
```
Comment (AndrielFR) [inline eduu/plugins/notes.py:259]: And here.

**Review by AndrielFR — APPROVED** [review]

File: eduu/plugins/notes.py (hunk 3 above)
Comment (iiiiii1wepfj) [reply eduu/plugins/notes.py:259]: @AndrielFR why?

Comment (AndrielFR) [reply eduu/plugins/notes.py:259]: Less code. 😂

**Comment by iiiiii1wepfj** [comment]
i think this is done

**Comment by iiiiii1wepfj** [comment]
i think this is good now

**Comment by alissonlauffer** [comment]
Looks like the workflow is broken because Nekobin expired the file, so I'll merge manually.

**Comment by AndrielFR** [comment]
> Looks like the workflow is broken because Nekobin expired the file, so I'll merge manually.

Oh, ok. 🤝

**Comment by alissonlauffer** [comment]
> @alissonlauffer You can merge it now?

Haven't you seen that it already got merged? xD

**Pull request merged**